### PR TITLE
MM New Ported Derived Table for MetadDB Instance publication

### PR DESCRIPTION
**Pull request by jenmawe**
This is a new ported over derived table for publication information in FOLIO instances.

### CODE REVIEW

File: sql_metadb/derived_tables/instance_publication.sql [hunk 1]
```diff
@@ -0,0 +1,32 @@
+-- This derived table extracts publication information from the FOLIO instance record
+DROP TABLE IF EXISTS instance_publication;
+
+CREATE TABLE instance_publication AS 
+SELECT 
+	it.id AS instance_id,
+	it.hrid as instance_hrid,
```
Comment (eliana-flo) [inline sql_metadb/derived_tables/instance_publication.sql:7]: Just for legibility/consistency, maybe make all your "AS" uppercase?

**Review by eliana-flo — APPROVED** [review]
All queries:
- [X] Pull request has a complete description
- [X] Query runs without errors
- [X] Query output is correct
- [X] Query logic is clear and well documented
- [X] Query is readable and properly indented
- [X] Table and column names are in all-lowercase
- [X] Quotation marks are used only where necessary

Derived tables:
- [X] Query begins with user documentation in comment lines
- [X] File name is listed in `runlist.txt` after dependencies
- [X] All columns have indexes
- [X] Table is vacuumed and analyzed

File: sql_metadb/derived_tables/instance_publication.sql (hunk 1 above)
Comment (jenmawe) [reply sql_metadb/derived_tables/instance_publication.sql]: Thanks for catching this @eliana-flo. I've made those changes and updated the PR.

**Review by lib-eric — APPROVED** [review]
Some odd publisher information in our test data but report is accurate!

All queries:
- [x] Pull request has a complete description
- [x] Query runs without errors
- [x] Query output is correct
- [x] Query logic is clear and well documented
- [x] Query is readable and properly indented
- [x] Table and column names are in all-lowercase
- [x] Quotation marks are used only where necessary

Derived tables:
- [x] Query begins with user documentation in comment lines
- [x] File name is listed in `runlist.txt` after dependencies
- [x] All columns have indexes
- [x] Table is vacuumed and analyzed

**Pull request merged**